### PR TITLE
PRODTASKS-688 change to fail the test when invoking from Jekins

### PR DESCRIPTION
**Pull request by rgdoliveira**

### CODE REVIEW

File: src/test/java/com/redhat/engineering/srcclr/SourceClearTest.java [hunk 1]
```diff
@@ -47,7 +48,7 @@ public void runSourceClear() throws Exception
         catch( Exception e )
         {
             logger.debug( "Caught exception executing the wrapper", e );
-            throw e;
+            Assert.fail();
```
Comment (rnc) [inline src/test/java/com/redhat/engineering/srcclr/SourceClearTest.java]: I  think perhaps we should use `Assert.fail(  "<message>" ) ` in order to propagate out the information. And maybe change that debug line to error?

**Comment by rnc** [comment]
And the tests need updating :-) 

Thanks for the contribution!

**Comment by rgdoliveira** [comment]
@rnc taking a better look in the code I think this change in "SourceClearTest.java" will not help.
Any exception that happens (failure when building the project or vulnerability found) will now be changed from Exception to test failure, right?

My idea was that any error in SourceClear binary execution (e.g project build) would throw an exception and any error in SourceClear Invoker tool (vulnerability found) would make the test to fail.

**Comment by rgdoliveira** [comment]
Just saw the variable "boolean exception = true" in SrcClrWrapper, maybe setting it to false can deal with this? Just not sure if there is a way to setup it to false in SourceClear Invoker parameters.

**Comment by rnc** [comment]
@rgdoliveira As it happens ScanException is only thrown if the scan fails. I think it would be better to refactor it to return that (or the underlying string) as a message so that its possible to distinguish between failure/errors. I'm having a look at the code now

**Comment by rnc** [comment]
Closing this in favour of #63

**Pull request closed without merging**